src/typescript/04-configuration/configuration.md

```diff
@@ -7,24 +7,25 @@
 
 ## Overview
 
-In this section we’ll learn about TypeScript configuration options to help us optimize our workflow.
+In this section we will:
 
-For the exercise, we will learn how to import and export functions between modules.
+- Learn about TypeScript configuration options to help us optimize our workflow
+- Learn about linting with TypeScript
 
 ## Configuring TypeScript
 
 We’re able to configure how our projects use TypeScript from selecting which files to compile to removing comments from those files. This is done through the creation of a `tsconfig.json` file.
 
 ### tsconfig.json
 
-Customize the TypeScript compiling and linting options by creating a `tsconfig.json` file in the root directory of your TypeScript project. The following config will
+Customize the TypeScript compiling and linting options by creating a `tsconfig.json` file in the root directory of our TypeScript project. The following config will
```

Comment:
We aren’t actually instructing them to do anything, so I would take the personal pronouns out of this completely.